code/game/gamemodes/vampire/vampire.dm

```diff
@@ -5,6 +5,21 @@
 	var/list/datum/mind/vampire_enthralled = list() //those controlled by a vampire
 	var/list/vampire_thralls = list() //vammpires controlling somebody
 
+
+/mob/living/carbon/human/Stat()
+	. = ..()
+	var/datum/antagonist/vampire/vamp = mind.has_antag_datum(/datum/antagonist/vampire)
+	if(vamp && statpanel("Status"))
+		stat("Total Blood", vamp.bloodtotal)
+		stat("Usable Blood", vamp.bloodusable)
+
+/mob/living/carbon/human/Life()
```

Comment:
Antag datums needing to call something on the host mob's `Life` is common - so adding a handler to `/datum/antagonist` that would be looped through per installed antag datum and called in a human's `Life` would be wise. Adding another implementation of `human/Life` should be avoided where possible.